### DISCUSSION TITLE
Deleting screenshot option

### DISCUSSION THREAD

**Discussion by sadrajafari** [scenerystack/community#35] [Q&A]
Hey, I am just wondering if I want to remove the option of screenshot how I can do that?

Thanks.

**Comment by sadrajafari** [comment]
I used Three. js in my code and now when I click on screenshot action, the created element with Three is not on the generated image. 

Is there any way I can fix this problem?

**Reply by brettfiedler** [reply]
Hi @sadrajafari, could you provide the code in question? It is difficult to tell if your problem is associated with SceneryStack code or Three.js without context. 

> Hey, I am just wondering if I want to remove the option of screenshot how I can do that?

Are you referring to the Screenshot option available in the navigation bar menu in [joist](https://github.com/phetsims/joist/blob/bb641a35f75e180c2ee8e4bc8bd829aea22ac8d8/js/PhetMenu.ts#L146-L187)?

**Comment by zepumph** [comment]
Hello there, and thanks for reaching out to us. I don't see a way to hide the screenshot Menu Item easily, but I do know that we have support for adapting THREE.js stages into our scenery scene graph, and I believe that we can get you to have your THREE code in the screenshot. 

We have a repository devoted to this integration called mobius. We use this in a simulation called [Density](https://phet.colorado.edu/sims/html/density/latest/density_all.html). For example [here](https://github.com/phetsims/density-buoyancy-common/blob/c7f6f0c3ce4ccfbf22c73be5f91cf403bed654e3/js/common/view/DensityBuoyancyScreenView.ts#L155) we use a `ThreeIsometricNode` to wrap the stage.

I'm not sure what your context is, and I'd be happy to assist more if you are able to provide more specifics about the help you need, but I would recommend trying to wrap your THREE stage into scenery to support the screenshot fully.
